introduction.md

```diff
@@ -6,27 +6,120 @@ title: Introduction
 # Introduction
 
 ## What is Foreman?
-Foreman is an open source project that helps system administrators manage servers throughout their lifecycle,
-from provisioning and configuration to orchestration and monitoring.
-Using Puppet, Chef, Salt, Ansible and Foreman's smart proxy architecture, you can easily automate repetitive tasks,
-quickly deploy applications, and proactively manage change, both on-premise with VMs and bare-metal or in the cloud.
 
-Foreman provides comprehensive, interaction facilities including a web frontend, CLI and RESTful API
-which enables you to build higher level business logic on top of a solid foundation.
+Foreman is an open source project that helps system administrators manage
+servers throughout their lifecycle, from provisioning and configuration to
+orchestration and monitoring. Provisioning support gives you easy control of
+setting up new servers, and using configuration management (Puppet, Ansible,
+Chef and Salt are supported), you can easily automate repetitive tasks. With
+Foreman, you can quickly deploy applications, and proactively manage change,
+both on-premise with VMs and bare-metal or in the cloud. Foreman scales well to
+multiple locations (offices, data centres, etc) and multiple organisations,
+allowing you to grow without losing your single source of infrastructure truth.
 
-Foreman is more than 7 years old, and deployed in many organizations, managing from 10s to 10,000s of servers.
+Foreman, provides comprehensive, interaction facilities including a web
+frontend, CLI and RESTful API which enables you to build higher level business
+logic on top of a solid foundation. It is deployed in many organizations,
+managing from 10s to 10,000s of servers. Several commercial products are based
+on Foreman.
 
-It is used in distributions such as RDO and RHOS (Red Hat OpenStack distribution) and has
-an [extensive library of plugins](/plugins).
-
-With Foreman you can:
+## High-level overview
 
 * Discover, provision and upgrade your entire bare-metal infrastructure
-* Create and manage instances across private and public clouds
+* Create and manage instances in virtualization environment and across private and public clouds
+* Install operating systems via PXE, local media or from templates or images
+* Control and gather reports from your configuration management software
 * Group your hosts and manage them in bulk, regardless of location
 * Review historical changes for auditing or troubleshooting
+* Web user interface, JSON REST API and CLI for Linux
 * Extend as needed via a robust plugin architecture
-* Automatically build images (on each platform) per system definition to optimize deployment
+
+## Notable features
+
+**Installation and usability**
+
+* **Easy POC installation**: With a dedicated one-command installer (which uses Puppet underneath), Foreman can be easily evaluated or customized as required.
```

Comment:
not sure if we need to highlight that the installer is puppet based, vs it can be easy to poc and easy to automate via yaml based answer file.

Reply:
OK